### PR TITLE
Change max-line-length to 120

### PR DESCRIPTION
**Pull request by guioliveirabh**
### Fixes

- #<number>

### Checklist

- [ ] Related tests were updated
- [ ] Related documentation was updated
- [ ] OpenAPI file was updated
- [ ] Run `tox`, no tests failed

### CODE REVIEW

**Comment by brhatiga** [comment]
PEP8 still mentions 79 chars max but I'm ok with extending it if the rest of the team votes to do so.

**Review by freedgit — DISMISSED** [review]
We have modern gear now.

**Comment by daniellutz** [comment]
I agree with the change, but if the team does not agree, maybe we could go for 100 chars?

File: tox.ini [hunk 1]
```diff
@@ -22,7 +22,7 @@ deps =
 commands = flake8 src bin migrations
 
 [flake8]
-max-line-length = 80
+max-line-length = 120
 select = C,E,F,W,B,B950,I
 ignore = W503,E501
```
Comment (fpob) [inline tox.ini:27]:
```suggestion
select = C,E,F,W,B,I
ignore = W503
```

To disable +10%.

**Review by thaorell — DISMISSED** [review]
👀

**Review by daniellutz — DISMISSED** [review]

**Review by thaorell — APPROVED** [review]

**Pull request merged**